markdown/org/docs/designs/waralee/options/knotinfront/en.md

```diff
@@ -2,5 +2,5 @@
 title: "Knot in Front"
 ---
 
-Normally the Waralee is closed with the tie in the front. This allows you to change
-that to a bow tie in the back.
\ No newline at end of file
+Normally the Waralee is closed with all straps tied in the front. This allows you to change
+that to a set tied in the front and in the back, the traditional way.
```

Comment:
If the knot in front option is set to "Knot in Back", the back ties are made longer instead, so both straps will be tied in the back. That contradicts this documentation. With knot in the back selected, both ties probably should be the same length or the option should be split into 3 (both ties at the front, both ties at the back, and the traditional way with shorter same-length straps).